.gitignore

```diff
@@ -20,3 +20,7 @@ clangs
 bin/test
 bin/unittest
 .DS_Store
+*.so.*
+*.so
+/bin/*
+*~
```

Comment:
It is to not add the temporary files. Many temporary files are created
which end wirh '~', to ignore them, i put it.
On Mar 19, 2016 1:36 AM, "jacob-carlborg" <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/jacob-carlborg/dstep/pull/57#discussion_r56714617>:
>
> > @@ -20,3 +20,7 @@ clangs
> >  bin/test
> >  bin/unittest
> >  .DS_Store
> > +*.so.*
> > +*.so
> > +/bin/*
> > +*~
>
> What is this for?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jacob-carlborg/dstep/pull/57/files/55b5fea4c98742241d615a882bb5032b2dfed00c#r56714617>
>